morgoth/base/middleware.py

```diff
@@ -0,0 +1,9 @@
+from time import sleep
+
+from django.conf import settings
+
+
+class SleepMiddleware(object):
+    def process_request(self, request):
+        if settings.DEBUG:
+            sleep(getattr(settings, 'SLEEP_SECONDS', 0))
```

Comment:
Alternatively, since you have to add this middleware via the settings anyways, you could just make it on all the time.